lib/store-hmac.js

```diff
@@ -0,0 +1,332 @@
+/*
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+const _ = require('lodash');
+const async = require('async');
+const bedrock = require('bedrock');
+const crypto = require('crypto');
+const database = require('bedrock-mongodb');
+const jwt = require('jsonwebtoken');
+const BedrockError = bedrock.util.BedrockError;
+
+// load config defaults
+require('./config');
+
+const logger = bedrock.loggers.get('app');
+
+// TODO: make responding to this event optional
+bedrock.events.on(
+  'bedrock-authn-did-jwt.config.keyStore', (strategy, callback) =>
+    strategy.registerStore(Store, callback));
```

Comment:
We should instantiate both an RSA Store and an HMAC Store here -- and create a single object that we pass to the strategy that will handle calling into either one. In other words, that we have two separate stores here is an implementation detail of *this* module, it's not something that the strategy should be getting involved with at all to manage for us. The strategy should expect a simple store object that they can ask to do things without thinking about it.